articles/tutorials/how-to-update-applications-client-secret.md

```diff
@@ -22,13 +22,15 @@ Here is an example of a command you can use to generate a random string:
 
 `dd if=/dev/random bs=48 count=1`
 
-## Using the Management API to set the new client_secret
+## Using the Management API to set the new Client Secret
 
-You can also change your application's `client_secret` by using [Update a client](/api/v2#!/Clients/patch_clients_by_id) in the Management API.  Set the `id` of the client you are updating, and then in the `body` section enter your new `client_secret` as a JSON field.
+You can change your client's secret by using [Rotate Secret API endpoint](/api/v2/clients/{id}/rotate-secret). This will rotate your (non encoded) secret with a secret generated by Auth0. This is the recommended approach, since we ensure a safe secret.
```

Comment:
small thing "This will rotate your (non encoded) secret"

it's going to rotate any kind  of secret (encoded or not), but the new secret will be non encoded.